src/grpcservermodule.cpp

```diff
@@ -195,8 +196,10 @@ void GRPCServerModule::shutdown() {
         return;
     state = ModuleState::STARTED_SHUTDOWN;
     SPDLOG_INFO("{} shutting down", GRPC_SERVER_MODULE_NAME);
+    std::chrono::time_point serverDeadline = std::chrono::system_clock::now() +
```

Comment:
Why this is required? Doesn't it affect tests time?

Reply:
I added this so we don't hang after shutdown if there are some open connections. With this change we will wait 5 seconds and then forcefully disconnect clients. If there are no active connections and we can shutdown immediately we will, so it won't extend testing time.

Reply:
Ok I get it now - Shutdown(int) is different than Shutdown(). First forcefully closes connection while second does not.